fastlane_core/spec/configuration_file_spec.rb

```diff
@@ -64,12 +64,6 @@
         expect(config[:another_boolean]).to be(true)
       end
 
-      it "ignores the same item being set after the first time" do
```

Comment:
By changing this, would this break setups where you have the same line in one config file twice, and you are used to them being ignored? Is there a way we can leave that test and make it pass, while still adding this new feature?